tests/figures/chart/charts_component.test.ts

```diff
@@ -647,6 +649,225 @@ describe("charts", () => {
     });
   });
 
+  test.each(["min", "max"])(
```

Comment:
nitpick: you could test both min/max in the same test rather than doing a test.each. I dont think it's less readable since the test are really short, and it would avoid mounting another spreadsheet component